.github/workflows/test-query-compiler-template.yml

```diff
@@ -1,4 +1,3 @@
-name: 'QC: integration tests'
```

Comment:
Why drop the name? Does this shorten it?

Reply:
It was removed by mistake. Restored.